test/wh_test_crypto.c

```diff
@@ -94,38 +94,19 @@ int whTest_CryptoClientConfig(whClientConfig* config)
     int res = 0;
     /* wolfcrypt */
     WC_RNG rng[1];
-    Aes aes[1];
-    RsaKey rsa[1];
     ecc_key eccPrivate[1];
     ecc_key eccPublic[1];
-    curve25519_key curve25519PrivateKey[1];
-    curve25519_key curve25519PublicKey[1];
-    Cmac cmac[1];
     uint32_t outLen;
     uint16_t keyId;
+    uint8_t iv[16];
     uint8_t key[16];
     uint8_t keyEnd[16];
     uint8_t labelStart[WH_NVM_LABEL_LEN];
     uint8_t labelEnd[WH_NVM_LABEL_LEN];
-    uint8_t iv[AES_IV_SIZE];
     char plainText[16];
     char cipherText[256];
     char finalText[256];
-    char cmacFodder[1000];
     uint8_t authIn[16];
-    uint8_t authTag[16];
-    uint8_t sharedOne[CURVE25519_KEYSIZE];
-    uint8_t sharedTwo[CURVE25519_KEYSIZE];
-    uint8_t knownCmacKey[] = {0x2b, 0x7e, 0x15, 0x16, 0x28, 0xae, 0xd2, 0xa6,
-        0xab, 0xf7, 0x15, 0x88, 0x09, 0xcf, 0x4f, 0x3c};
-    uint8_t knownCmacMessage[] = {0x6b, 0xc1, 0xbe, 0xe2, 0x2e, 0x40, 0x9f,
-        0x96, 0xe9, 0x3d, 0x7e, 0x11, 0x73, 0x93, 0x17, 0x2a, 0xae, 0x2d, 0x8a,
-        0x57, 0x1e, 0x03, 0xac, 0x9c, 0x9e, 0xb7, 0x6f, 0xac, 0x45, 0xaf, 0x8e,
-        0x51, 0x30, 0xc8, 0x1c, 0x46, 0xa3, 0x5c, 0xe4, 0x11, 0xe5, 0xfb, 0xc1,
-        0x19, 0x1a, 0x0a, 0x52, 0xef, 0xf6, 0x9f, 0x24, 0x45, 0xdf, 0x4f, 0x9b,
-        0x17, 0xad, 0x2b, 0x41, 0x7b, 0xe6, 0x6c, 0x37, 0x10};
-    uint8_t knownCmacTag[] = {0x51, 0xf0, 0xbe, 0xbf, 0x7e, 0x3b, 0x9d, 0x92,
-        0xfc, 0x49, 0x74, 0x17, 0x79, 0x36, 0x3c, 0xfe};
```

Comment:
I'm moving each of these out to individual functions as we go, so there will only be a few common variables here.  The rule was to support C89, but we are clearly C99 in wolfHSM so we can be a bit more lax.  I still tend to slide things to the top because it's a good way to show me my functions are too long when the list is large or not coherent.  You'll see this list get smaller with each updated algo.